geomstats/visualization.py

```diff
@@ -373,7 +386,8 @@ def convert_to_trihedron(point, space=None):
     return trihedrons
 
 
-def plot(points, ax=None, space=None, **point_draw_kwargs):
+def plot(points, ax=None, space=None,
+         point_type='extrinsic', **point_draw_kwargs):
```

Comment:
`point_type=None` here, and then have a line:
if point_type is None:
    point_type = 'extrinsic'

in the code specific to hyperbolic space